### ISSUE TITLE
Final extra zoom level to double pixel scaling of finest map

### ISSUE THREAD

**Issue by sbridger** [dcaliste/maep-qt#36]
I often have to locate very close.
It would be good to have a final zoom level that doubles-up the pixels of the last (finest) map to get 2x apparent resolution (i.e. map will only show 1/4 of area.

Again this is partly a function of poor eyesight

**Comment by dcaliste** [comment]
Thank you for using Mæp. There's already an option in the parameters to always draw the map in double pixel sizes.

Besides, your remark about having it as an additional zoom level when in maximum is a nice one. I'll try to see if I can fit the implementation in my schedule !

**Comment by sbridger** [comment]
Yes it was double up mode, which I imagined would work as I described...
